CI: build fails - PyInstaller issue?

```
...
   ModuleNotFoundError: No module named 'PyInstaller.__main__'; 'PyInstaller' is not a package
  [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

Encountered error while generating package metadata.

See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
Error: Process completed with exit code 1.
```

see for example https://github.com/mandiant/capa/actions/runs/2090953281

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -35,10 +35,10 @@ jobs:
           python-version: 3.8
       - if: matrix.os == 'ubuntu-18.04'
         run: sudo apt-get install -y libyaml-dev
-      - name: Install PyInstaller
+      - name: Upgrade pip, setuptools
         run: pip install --upgrade pip setuptools
       - name: Install PyInstaller
-        run: pip install 'pyinstaller==4.2'
+        run: pip install 'pyinstaller==4.10'
       - name: Install capa
         run: pip install -e .
       - name: Build standalone executable

